nuitka/OptionParsing.py

```diff
@@ -1390,14 +1390,16 @@
     "--windows-console-mode",
     action="store",
     dest="console_mode",
-    choices=("force", "disable", "attach"),
+    choices=("force", "disable", "attach", "hide"),
```

Comment:
In a moment of bike shedding, I would like the name it `minimize_after_start` or such, or anything that conveys it better that it cannot start hidden.

Reply:
I will stick to `hide` for these options, actual reading will always be required. At least hide is a more obvious name than attach.